subprojects/common/elastic/src/main/kotlin/com/avito/android/elastic/ElasticClientFactory.kt

```diff
@@ -0,0 +1,43 @@
+package com.avito.android.elastic
+
+import com.avito.time.DefaultTimeProvider
+import com.avito.time.TimeProvider
+import okhttp3.OkHttpClient
+import java.util.concurrent.TimeUnit
+
+object ElasticClientFactory {
+
+    private const val defaultTimeoutSec = 10L
+
+    private val timeProvider: TimeProvider by lazy { DefaultTimeProvider() }
```

Comment:
Is it expensive to make it lazy?

Reply:
nope, in fact it maybe more expensive to make it lazy here; i'll fix it